src/test/java/com/aws/greengrass/ipc/IPCEventStreamServiceTest.java

```diff
@@ -83,6 +83,7 @@ public void setup() throws UnauthenticatedException {
         ipcEventStreamService = new IPCEventStreamService(mockKernel, new GreengrassCoreIPCService(), config,
                 mockAuthenticationHandler);
         ipcEventStreamService.startup();
+        Thread.sleep(5000);
```

Comment:
Do we need a delay this long?

Reply:
Not likely, but this does appear to fix it. We can retry with shorter times too in a different PR. If this truly does resolve the issue, we may want to also add a delay in the real IPC server so that we don't hit timing issues like this for the customer.